website/docs/reference/commands/compile.md

```diff
@@ -33,13 +33,42 @@ dbt compile --select stg_payments
 dbt compile --inline "select * from {{ ref('raw_orders') }}"
 ```
 
-<Lightbox src="/img/docs/reference/dbt-compile.png" title="dbt compile --select stg_payments"/>
+returns the following:
+
+
+```bash
+dbt compile --select stg_orders
```

Comment:
should this be ` --select stg_payments`? instead per example above this section? 

```
dbt compile --select stg_payments
dbt compile --inline "select * from {{ ref('raw_orders') }}"

```

Reply:
so should the example change to `stg_orders`?

Reply:
Having the basic demo and the worked example reference the same things isn't the worst idea! Feel free to change it 👍